src/blocks/recommendations/components/parsely-recommendations-inspector-controls.js

```diff
@@ -0,0 +1,183 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { InspectorControls } from '@wordpress/block-editor';
+import {
+	PanelBody,
+	PanelRow,
+	RadioControl,
+	RangeControl,
+	SelectControl,
+	TextControl,
+	ToggleControl,
+} from '@wordpress/components';
+
+const ParselyRecommendationsInspectorControls = ( {
+	attributes: { boost, imagestyle, limit, showimages, sort, tag, title },
+	setAttributes,
+} ) => {
+	return (
+		<InspectorControls>
+			<PanelBody title="Settings" initialOpen={ true }>
+				<PanelRow>
+					<TextControl
+						label={ __( 'Title', 'wp-parsely' ) }
+						value={ title }
+						onChange={ ( newval ) => setAttributes( { title: newval } ) }
+					/>
+				</PanelRow>
+				<PanelRow>
+					<RangeControl
+						label={ __( 'Maximum Results', 'wp-parsely' ) }
```

Comment:
```suggestion
						label={ __( 'Maximum Number Of Recommendations', 'wp-parsely' ) }
```

Might be a bit too wordy, but trying to make it more contextual for the user.

Reply:
I get the intention. IMO "results" is clear enough and widely used. But I'll keep this open just in case.

Reply:
+1 on @acicovic's point.

Reply:
OK, marking this as resolved but feel free to come back if needed.